src/app/common/hooks/use-element-height-listener.ts

```diff
@@ -0,0 +1,22 @@
+import { useEffect } from 'react';
```

Comment:
This seems cool and maybe useful to me for dynamic height sizing which has proved tricky in containers.

I was trying to use calculated heights via variables which `panda` doesn't like and had considered using `style` so this is a good idea